### PR TITLE
Fixed print issue

### PR DESCRIPTION
**Pull request by aliang8**
Fixed print to display parentheses around exponents. Added appropriate test cases.

### CODE REVIEW

File: lib/print.js [hunk 1]
```diff
@@ -62,7 +62,21 @@ const printApply = (node, parent) => {
...
         return result;
     } else if (op === 'pow') {
-        return `${print(args[0], node)}^${print(args[1], node)}`;
+        const exponent = node.args[1];
+        let bool = false;
+        // check if exponent has multiple terms and if the exponent
+        // contains a div/mul operation
+        // e.g x^((x + 1)/(2 + 2))
+        if (exponent.args){
+            if (exponent.op == 'div' || exponent.op == 'mul'){
```
Comment (k4b7) [inline lib/print.js]: There should be a space between conditionals the opening brace.  See https://github.com/semantic-math/math-parser/issues/37.  It can't be handled in a separate diff.

Comment (evykassirer) [reply lib/print.js]: I think you could probably add that to the linter

File: lib/print.js [hunk 2]
```diff
@@ -62,7 +62,21 @@ const printApply = (node, parent) => {
...
+                bool = true;
+            }
+        }
+        if (bool) {
```
Comment (k4b7) [inline lib/print.js]: Can you change this to `if (isMul(exponent) || isDiv(exponent) {` and define those helper functions as necessary.  Then you can get rid of the comment above b/c the intent is clear from the code.

**Review by k4b7 — CHANGES_REQUESTED** [review]
Tests look great.  Requesting a minor change in print.js.

File: package.json [hunk 3]
```diff
@@ -25,6 +25,8 @@
     "babel-plugin-transform-object-rest-spread": "^6.22.0",
     "babel-preset-es2015": "^6.22.0",
     "eslint": "^3.15.0",
+    "eslint-config-google": "^0.7.0",
+    "eslint-plugin-sort-requires": "^2.1.0",
```
Comment (k4b7) [inline package.json]: Please leave this out.  I'm open to changing lint set up for this repo but we should discuss it first and do it in a separate PR.

**Review by k4b7 — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (aliang8) [reply package.json]: Yup, fixed!

File: .eslintrc [hunk 4]
```diff
@@ -1,18 +0,0 @@
...
-    "rules": {
-        "no-constant-condition": 0
-    },
-    "env": {
-        "browser": true,
-        "es6": true,
-        "node": true,
-        "mocha": true
-    }
-}
```
Comment (k4b7) [inline .eslintrc]: This was there before the diff and shouldn't be removed.

File: package.json [hunk 5]
```diff
@@ -24,8 +24,6 @@
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.22.0",
     "babel-preset-es2015": "^6.22.0",
-    "eslint": "^3.15.0",
-    "eslint-plugin-flowtype": "^2.30.0",
```
Comment (k4b7) [inline package.json]: This shouldn't have been removed either.

**Comment by k4b7** [comment]
@aliang8 one last change before this can be merged – please revert changes to package.json.

**Comment by k4b7** [comment]
@aliang8 for an easy way to revert files to a previous commit see http://stackoverflow.com/questions/215718/reset-or-revert-a-specific-file-to-a-specific-revision-using-git.

**Comment by k4b7** [comment]
Looks good.  Thanks for fixing this.

**Review by k4b7 — APPROVED** [review]
